.pre-commit-config.yaml

```
@@ -0,0 +1,6 @@
+repos:
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  rev: v0.11.11
+  hooks:
+    - id: ruff-check
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)